java/fury-core/src/main/java/org/apache/fury/resolver/DisallowedList.java

```diff
@@ -24,15 +24,25 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
 import java.util.Set;
+import java.util.TreeSet;
 import java.util.stream.Collectors;
 import org.apache.fury.exception.InsecureException;
 
 /** A class to record which classes are not allowed for serialization. */
 class DisallowedList {
   private static final String DISALLOWED_LIST_TXT_PATH = "fury/disallowed.txt";
+  // when disallowed.txt changed, update this hash by print the result of `calculateSHA256`
+  private static final String SHA256_HASH =
+      "d418999c49b0aa83b8bde8a79aa758a16adb5599e384f842db65dbcd633c541b";
```

Comment:
could this be calculated by the build process and this class generated?

Reply:
* it's also probably true that if the build fails after fury/disallowed.txt is modified (without the SHA upgrade) then that should be enough
* could we remove the main method just have the a unit test that compares the generated SHA to the hardcoded one?
* main methods in classes is not usually regarded as good
* the test could be run instead to get the new SHA, so a coder could grab that and updated the hardcoded SHA

Reply:
I removed the main method. The chekc happs when class init. If check failed, the class init will fail, existing tests in DisallowedListTest will all fail too. Seems adding new tests is unnecesary @pjfanning 

caculate the hash at build process will make the build complex. Since this is not updated frequently, how about updating it manually every time?

Reply:
Sure